main_test.go

```diff
@@ -786,11 +787,17 @@ func TestWithTransportSerializer(t *testing.T) {
 				Type:  wire.Call,
 				Value: wire.NewValueStruct(wire.Struct{}),
 			}),
+			wantTransportHeaders: map[string]string{
+				"RPC-Thrift-Envelope": "true",
```

Comment:
nit: pretty sure that the canonical form of the header key (as defined by net/http) is `Rpc-Thrift-Envelope`.

You could also just use https://golang.org/pkg/net/http/#CanonicalHeaderKey